go.mod

```diff
@@ -50,6 +50,8 @@ require (
 	gopkg.in/mcuadros/go-syslog.v2 v2.3.0
 )
 
+require github.com/prometheus/procfs v0.7.3
```

Comment:
Note that we could take a smaller portion of this package (we only need the `/proc/mdstat` handling), but this could actually be interesting to switch to in the future - gopsutils is pretty large, and this is a more targeted package for gathering Linux system statistics from /proc and /sys.